ceph_iscsi_config/settings.py

```diff
@@ -14,6 +14,59 @@ def init():
     config = Settings()
 
 class Settings(object):
+    GATEWAY_SETTINGS = [
+        "dataout_timeout",
+        "nopin_response_timeout",
+        "nopin_timeout",
+        "cmdsn_depth",
+        "immediate_data",
+        "initial_r2t",
+        "max_outstanding_r2t",
+        "first_burst_length",
+        "max_burst_length",
+        "max_recv_data_segment_length",
+        "max_xmit_data_segment_length"]
+
+    LIO_YES_NO_SETTINGS = ["immediate_data", "initial_r2t"]
```

Comment:
Why is one LIO and one is GATEWAY? I didn't get reason for the naming prefix difference.

Reply:
I was using “GATEWAY” to signify target / gateway-level overrides (vs disk, initiator, etc). I just used “LIO” to signify that LIO wants Yes/No (not necessarily tied to gateway settings).